test/common/stats/BUILD

```diff
@@ -162,6 +162,16 @@ envoy_cc_fuzz_test(
     ],
 )
 
+envoy_cc_fuzz_test(
+    name = "utility_fuzz_test",
+    srcs = ["utility_fuzz_test.cc"],
+    corpus = "utility_corpus",
+    deps = [
+        "//source/common/stats:isolated_store_lib",
```

Comment:
Is this dep needed?

Reply:
No, just tested locally and the fuzzer builds and runs properly without it. I thought it might be necessary for some of the other stats utility functions, though - should I remove it?

Reply:
If the fuzzer builds and runs without it, yes, please remove it. The `utility_lib` target should declare all the dependencies it needs, and Bazel knows to pull those in when compiling this fuzz target.

Reply:
Fixed in the latest commit.